src/test/java/org/assertj/core/api/classes/ClassAssert_hasPackage_String_Test.java

```diff
@@ -0,0 +1,28 @@
+package org.assertj.core.api.classes;
+
+import static org.mockito.Mockito.verify;
+
+import org.assertj.core.api.ClassAssert;
+import org.assertj.core.api.ClassAssertBaseTest;
+import org.junit.jupiter.api.DisplayName;
+
+/**
+ * Tests for <code>{@link ClassAssert#hasPackage(String)}</code>.
+ *
+ * @author Matteo Mirk
+ */
+@DisplayName("Classes hasPackage")
+class ClassAssert_hasPackage_String_Test extends ClassAssertBaseTest {
```

Comment:
```suggestion
class ClassAssert_hasPackage_with_String_Test extends ClassAssertBaseTest {
```

Reply:
I tried to follow the other tests convention and didn't see "with" used in the name, am I wrong?